lab-dustin/lib/readfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const { readFile } = require('fs');
+
+exports.readData = function(file, callback) {
+  readFile(`${__dirname}/../assets/${file}`, function(err, buff){
+    if (err) { callback(err); }
+    console.log(file);
+    console.log(buff.toString('hex').slice(0,8));
+    if(callback) { callback(null,
+      {
+        hex: buff.toString('hex').slice(0,8),
+        filepath: `${__dirname}/../assets/${file}`,
```

Comment:
This `filepath` calculation is repeated - I'd probably define it once before calling `readfile()`.